mindsdb/integrations/handlers/pgvector_handler/pgvector_handler.py

```diff
@@ -30,8 +32,42 @@ class PgVectorHandler(VectorStoreHandler, PostgresHandler):
     def __init__(self, name: str, **kwargs):
 
         super().__init__(name=name, **kwargs)
+        self._is_shared_db = False
         self.connect()
 
+    def _make_connection_args(self):
+        cloud_pgvector_url = os.environ.get('KB_PGVECTOR_URL')
+        if cloud_pgvector_url is not None:
+            result = urlparse(cloud_pgvector_url)
+            self.connection_args = {
+                'host': result.hostname,
+                'port': result.port,
+                'user': result.username,
+                'password': result.password,
+                'database': result.path[1:]
+            }
+            self._is_shared_db = True
+        return super()._make_connection_args()
+
+    def get_tables(self) -> Response:
+        # Hide list of tables from all users
+        if self._is_shared_db:
+            return Response(RESPONSE_TYPE.OK)
```

Comment:
From user perspective: database exists but they can't see tables inside (because it returns nothing). If it raised exception they would have more questions and think that something doesn't work